openeo/metadata.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 import logging
+import pystac
```

Comment:
if pystac is imported here at top level like this, it has to be declared as global dependency in setup.py,
(not just as test dependency like we did originally)